apps/web/src/utils/proofs/sybil_resistance.ts

```diff
@@ -163,11 +166,12 @@ export async function sybilResistantUsernameSigning(
       };
     }
 
+    const expirationTimeUnix = Math.floor(Date.now() / 1000) + parseInt(EXPIRY);
```

Comment:
make sure that expiration is set to a time in the future, determined by the expiration in seconds env var